### PR TITLE
Add coq-freespec-core.0.3, coq-freespec-ffi.0.3, coq-freespec-exec.0.3

### PR DESCRIPTION
**Pull request by lthms**

### CODE REVIEW

File: released/packages/coq-freespec-ffi/coq-freespec-ffi.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "coq" {>= "8.12" & < "8.14~"}
+  "coq-freespec-core" {= "0.3"}
+  "coq-coqffi" {= "1.0.0~beta5"}
+  "coq-simple-io" {>= "1.0" & < "2.0~"}
+]
+
+tags: [
+  "keyword:foreign function interface"
+  "category:Miscellaneous/Coq Extensions"
+  "logpath:FreeSpec.Exec"
```
Comment (clarus) [inline released/packages/coq-freespec-ffi/coq-freespec-ffi.0.3/opam:34]: This is the same path as `coq-freespec-exec`. I wonder if there is a typo.

File: released/packages/coq-freespec-core/coq-freespec-core.0.3/opam [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  "coq" {>= "8.12" & < "8.14~"}
+  "coq-ext-lib" {>= "0.11.2"}
+]
+
+tags: [
+  "keyword:effects"
+  "keyword:freer"
+  "keyword:program logic"
+  "category:Mathematics/Category Theory"
+  "logpath:FreeSpec.Core"
```
Comment (clarus) [inline released/packages/coq-freespec-core/coq-freespec-core.0.3/opam:36]: Ideally there should be a `date:yyyy-mm-dd` field too.

File: released/packages/coq-freespec-ffi/coq-freespec-ffi.0.3/opam (hunk 1 above)
Comment (lthms) [reply released/packages/coq-freespec-ffi/coq-freespec-ffi.0.3/opam]: Fixed, thanks!

File: released/packages/coq-freespec-core/coq-freespec-core.0.3/opam (hunk 2 above)
Comment (lthms) [reply released/packages/coq-freespec-core/coq-freespec-core.0.3/opam:37]: Should be good

**Pull request merged**